### PR TITLE
Updated vue pwa post to get rid of implicit and use latest versions

### PR DESCRIPTION
**Pull request by dogeared**
This is an update of the post to support the pkce flow rather than implicit. Also, some changes to the way the latest version of vue router works.

### CODE REVIEW

File: _source/_posts/2019-05-30-vue-pwa-typescript.md [hunk 1]
```diff
@@ -357,35 +364,35 @@ In many situations, you will want to restrict access to parts of your applicatio
...
+Once you have finished the registration process, you will be taken to the developer dashboard. Each application that you want to use with Okta authentication must be registered and will receive its own client ID. Click on **Add Application** and, on the next screen, select **Single Page Application**. When you click on **Next** you will see a screen with settings. Change the default value for **Login redirect URIs** to `http://localhost:8080/callback`. You can leave all the other defaults. After clicking **Done**, scroll down and make note of the **client_id** value. You'll need it shortly.
 
 {% img blog/vue-pwa/create-new-app.png alt:"Create New Okta Application" width:"800" %}{: .center-image }
 
-Once you are finished you will be given a `clientId`. This is needed in your application when configuring Okta. In your application directory now run the following command.
+In your application directory now run the following command.
 
 ```bash
-npm i @okta/okta-vue@1.1.0 @types/okta__okta-vue@1.0.2
+npm i @okta/okta-vue @types/okta__okta-vue
```
Comment (mraible) [inline _source/_posts/2019-05-30-vue-pwa-typescript.md:374]: I would specify versions here - that way when something breaks between major releases, this post will still work.

File: _source/_posts/2019-05-30-vue-pwa-typescript.md [hunk 2]
```diff
@@ -308,7 +315,7 @@ This component simply shows the book's details obtained from the route's query p
 
 When the application was generated by the `vue` command line tool, it also created a `HelloWorld` component at `src/components/HelloWorld.vue`. This is not needed in the application, so you can delete it. If you delete this file, you'll need to delete references to it in `src/views/Home.vue` as well.
 
-In order for a sub-component to be shown, it must be registered with the router. Open `src/router.ts` and replace it with the code below.
+In order for a sub-component to be shown, it must be registered with the router. Open `src/router/index.ts` and replace it with the code below.
```
Comment (mraible) [inline _source/_posts/2019-05-30-vue-pwa-typescript.md:318]: `src/router.ts` is the correct filename: https://github.com/oktadeveloper/okta-vue-books-pwa-example/blob/master/src/router.ts

Comment (dogeared) [reply _source/_posts/2019-05-30-vue-pwa-typescript.md:318]: that's not how the latest version of the cli builds it

Comment (mraible) [reply _source/_posts/2019-05-30-vue-pwa-typescript.md:318]: In that case, the example needs an update too. :)

Comment (dogeared) [reply _source/_posts/2019-05-30-vue-pwa-typescript.md:318]: done!

**Comment by dogeared** [comment]
I added the changelog and rebased master, so it's ready to merge.

**Pull request merged**